src/fileInput/readerInfo.cpp

```diff
@@ -28,14 +28,14 @@ using namespace readerConfig;
 
 void basicReaderInfo::setFlag (int flagID)
 {
-    if (flagID < 32)
+    if (flagID < 32 && flagID > 0)
```

Comment:
should be >=0,  0 is a valid bit

Reply:
For some reason I was thinking `1 << 0 == 0`...

Reply:
Fixed with bdad756